docs/console/company-configuration/providers/extension-providers.md

```diff
@@ -0,0 +1,40 @@
+---
+id: extension-providers
+title: Extension Providers
+sidebar_label: Extension Providers
+---
+
+:::info
+
+Extension Providers can only be used when the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled.
+
+:::
+
+Some providers can extend or replace specific Console features.
+
+The steps to enable a Project Extension, using an Extension Provider, are the following:
+
+- check that the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled for the project; 
+- connect a _capable Provider_ (more details later) to its Company, as described in the [Management page](/console/company-configuration/providers/configure-provider.mdx);
+- navigate to [_Project Settings_](/console/project-configuration/project-settings.md) and click on the _Providers_ tab;
+- enable the desired Extension in the _Extensions_ section, selecting a suitable Provider.
+
+![Extension Provider banner](img/extension-provider-banner.png)
+
+An Extension is characterized by a Provider capability:
+
+- **only one capability** is associated to a specific Extension;
```

Comment:
this is not true, we currently have only one capability and the current interface provides support for a single URL but we may support multiple capabilities with multiple specific urls in the future

Reply:
changed this part to make clear that one capability from the Extension API Provider type corresponds to one extension